docs/manual/docs/cmd/spo/site/site-list.md

```diff
@@ -15,6 +15,7 @@ Option|Description
 `--help`|output usage information
 `--type [type]`|type of modern sites to list. Allowed values `TeamSite|CommunicationSite`, default `TeamSite`
 `-f, --filter [filter]`|filter to apply when retrieving sites
+`--deleted [deleted]`|use this switch to only return deleted sites
```

Comment:
As we are not passing anything into this option we can just use `--deleted` and remove `[deleted]`